python/cudf/cudf/core/column/column.py

```diff
@@ -331,6 +337,53 @@ def to_array(self, fillna=None):
         """
         return self.to_gpu_array(fillna=fillna).copy_to_host()
 
+    def _fill(self, fill_value, begin=0, end=-1, inplace=False):
+
+        if begin < 0:
+            begin += self.size
+
+        if end < 0:
+            end += self.size
+
+        begin = max(0, min(self.size, begin))
+        end = max(0, min(self.size, end))
+
+        if begin < 0 or end <= begin or begin >= self.size:
+            return self if inplace else self.copy()
```

Comment:
It's impossible for `begin` to be `< 0` now
```suggestion
        if end <= begin or begin >= self.size:
            return self if inplace else self.copy()
```